drivers/dma/Kconfig.siwx917

```diff
@@ -12,3 +12,8 @@ config DMA_SILABS_SIWX917_COMMON_INIT_PRIORITY
 	int "Common initialization priority"
 	depends on DMA_SILABS_SIWX917
 	default 42
+
+config SG_BUFFER_COUNT
+	int "The maximum allowable number of buffers for scatter-gather transfers"
+	depends on DMA_SILABS_SIWX917
```

Comment:
As soon as you have two or more symbols with the same single dependency, it typically makes sense to remove the `depends on` and put the options behind the same `if ... endif` block (this has the same effect as an explicit `depends on` for all options inside the block)